projects/pt1/e2e_testing/xfail_sets.py

```diff
@@ -2636,6 +2637,7 @@
     "ViewNoChange1dModule_basic",
     "ViewNoChange2dModule_basic",
     "ViewNoChange3dModule_basic",
+    "WeightNormInterfaceModule_basic",
```

Comment:
This should be added to FX_IMPORTER_XFAIL_SET.

Reply:
```"WeightNormInterfaceModule_basic",``` does appear under FX_IMPORTER_XFAIL_SET too, so the FX_Importer tests should be covered.

Reply:
(see line 476 in xfail_sets.py)